src/components/markdown-renderer/regex-plugins/replace-legacy-pdf-short-code.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import markdownItRegex from 'markdown-it-regex'
+import MarkdownIt from 'markdown-it/lib'
+
+const finalRegex = /^{%pdf (.*) ?%}$/
+
+export const legacyPdfShortCode: MarkdownIt.PluginSimple = (markdownIt) => {
+  markdownItRegex(markdownIt, {
+    name: 'legacy-pdf-short-code',
+    regex: finalRegex,
+    replace: (match: string) => {
+      return `<a target="_blank" rel="noopener noreferrer" href="${match}">${match}</a>`
```

Comment:
Don't we have an external Link component for this?

Reply:
This is a markdown it plugin, not a replacer. So I don't know what you mean.

Reply:
You're right, my bad.